module/newrelic/composer.json

```diff
@@ -0,0 +1,42 @@
+{
+    "name": "ergonode/newrelic",
+    "type": "ergonode-module",
+    "description": "Ergonode - New Relic",
+    "homepage": "https://ergonode.com",
+    "license": "OSL-3.0",
+    "config": {
+        "bin-dir": "bin",
+        "preferred-install": {
+            "*": "dist"
+        },
+        "sort-packages": true
+    },
+    "require": {
+        "php": "^7.4",
```

Comment:
I added it here for bigger flexibility https://github.com/ergonode/backend/pull/979/files/4a053e31062433b36c4932b4d118566405c70cde#diff-b91a11994dd8409b88a2fbd4e50f0dab529be2e43225fb3e27b430544cc69874R28
Otherwise, it would be necessary for each environment and you might not want it in some. What do you think?

Reply:
that's ok